.github/workflows/push-docker-image-debug.yaml

```diff
@@ -0,0 +1,73 @@
+name: publish docker image
```

Comment:
Why do we need this?

Reply:
To iterate with concepts like this much quicker when we need to validate how the codebase interacts with deployments.

Reply:
I just found an issue with the deployed debug image that would have ended up as a separate PR / review / main / merge loop otherwise!

Reply:
I intend to introduce this file as part of a separate PR though, I just needed this idea to quickly iterate through images while testing. Removed from the PR!